dashboards/grafana-dashboard-insights-compliance-general.template.yaml

```
@@ -1488,12 +1488,15 @@ objects:
             "targets": [
               {
                 "datasource": {
-                  "uid": "$datasource"
+                  "type": "prometheus",
+                  "uid": "${cloudwatch_datasource}"
                 },
-                "expr": "sum(kafka_consumergroup_group_topic_sum_lag{topic=~\".*platform.inventory.events\", group=\"complianceinventory-events-consumer\"}) by (topic)",
+                "editorMode": "code",
+                "expr": "sum by(topic) (aws_kafka_max_offset_lag_sum{consumer_group=\"complianceinventory-events-consumer\", topic=~\".*platform.inventory.events\"})",
```

According to the latest update, this should be changed to

```suggestion
                "expr": "sum by(topic) (aws_kafka_sum_offset_lag_sum{consumer_group=\"complianceinventory-events-consumer\", topic=~\".*platform.inventory.events\"})",
```